### ISSUE TITLE
User Timezone

### ISSUE THREAD

**Issue by zek** [techouse/intl-date-time#9]
Thank you for this great package!

I noticed a missing feature that original datetime field has.

This package doesn't handle time zone.

~~~
        Nova::userTimezone(function () {
            return 'Europe/Istanbul';
        });
~~~

**Comment by techouse** [comment]
Explain how you would implement a timezone with this plugin. I am asking because this plugin is here help users deal with locales, not timezones.

Otherwise try and use `Europe/Athens` which is basically the same timezone.

**Comment by techouse** [comment]
Aha, I see the issue. Will get right on it. ✈️

**Comment by techouse** [comment]
Implemented in [v1.2.6](https://github.com/techouse/intl-date-time/commit/c6ca3409aba535e3be5c6b186ed6326271fc7baf). Thanks for reporting! 🙏
